src/components/Calculator.js

```diff
@@ -1,61 +1,54 @@
-import { PureComponent, Fragment } from 'react';
+import React, { useState, useEffect } from 'react';
 import Button from './Button';
 import './Cal.css';
 import calculate from '../logic/calculate';
 
-class Calculator extends PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.state = {
+const Calculator = () => {
+  const [obj, setObj] = useState({});
```

Comment:
- [ OPTIONAL] Kindly always use a  camelCase notation for the function component instead of Pascal case notation you can visit [js naming convention](https://www.w3schools.com/js/js_conventions.asp)